packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -336,12 +333,26 @@ const start = async zcf => {
     );
   };
 
+  const LiquidateProposalShape = M.splitRecord({
+    give: {
+      In: AmountShape, // TODO brand-specific AmountShape
```

Comment:
I first read this as some sort of parameter to AmountShape. Could be more explicit:
```suggestion
      In: AmountShape, // TODO get amount shape from brand
```

When do you think that should be done?

Reply:
Done.

> When do you think that should be done?

No strong opinion. If you have any to suggest for this PR, that'd be great. But even with these left for later, this PR is still strictly stricter than the status quo.

When do you think that should be done?

Reply:
probably after Vaults Release. though I might slip it into making vaultFactory upgradeable. (might help the factoring to bear this requirement in mind)